### PR TITLE
Migrate gRPC module with protobuf package changes

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Purpose
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/1635
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [x] Added tests

### CODE REVIEW

**Review by BuddhiWathsala — DISMISSED** [review]

**Review by daneshk — DISMISSED** [review]

File: ballerina-tests/tests/06_server_streaming_client.bal [hunk 1]
```diff
@@ -14,6 +14,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerina/grpc;
 // This is client implementation for server streaming scenario
```
Comment (dilanSachi) [inline ballerina-tests/tests/06_server_streaming_client.bal:18]: Can you remove this line? I have missed it when transfering the tests.

File: ballerina-tests/tests/25_return_data_streaming_client.bal [hunk 2]
```diff
@@ -14,18 +14,19 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerina/grpc;
 // This is client implementation for server streaming scenario
```
Comment (dilanSachi) [inline ballerina-tests/tests/25_return_data_streaming_client.bal:18]: This line also

File: ballerina-tests/tests/06_server_streaming_client.bal (hunk 1 above)
Comment (MadhukaHarith92) [reply ballerina-tests/tests/06_server_streaming_client.bal]: Removed in 22fe1d0

File: ballerina-tests/tests/25_return_data_streaming_client.bal (hunk 2 above)
Comment (MadhukaHarith92) [reply ballerina-tests/tests/25_return_data_streaming_client.bal]: Removed in 22fe1d0

**Review by dilanSachi — APPROVED** [review]

**Review by BuddhiWathsala — APPROVED** [review]

**Pull request merged**